src/core/etl/src/Flow/ETL/UniqueFactory.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL;
+
+class UniqueFactory
+{
+    public static function int(int $min, int $max) : int
```

Comment:
Maybe also:
```suggestion
    private function __construct()
    {}

    public static function int(int $min, int $max) : int
```
Cause this class has only static methods.